modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -3,35 +3,54 @@
 // * scalability_and_performance/ztp-deploying-disconnected.adoc
 
 :_content-type: PROCEDURE
-[id="talo-platform-prepare-for-update_{context}"]
-= Preparing for the updates
+[id="talo-platform-prepare-end-to-end_{context}"]
+= End-to-end procedures for updating clusters in a disconnected environment
 
-If both the Hub and the spoke clusters are running {product-title} 4.9, you need to update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
+[id="talo-platform-prepare-for-update_{context}"]
+== Preparing for the updates
 
-For more information about how to update ZTP, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-upgrading-gitops-ztp_ztp-deploying-disconnected[Upgrading GitOps ZTP].
+If both the hub and the spoke clusters are running {product-title} 4.9, you need to update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
 
 [id="talo-platform-prepare-for-update-env-setup_{context}"]
 == Setting up the environment
 
-If you deployed ZTP in a disconnected environment as instructed in the xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-acm-preparing-to-install-disconnected-acm_ztp-deploying-disconnected[Preparing the disconnected environment] section, you need to do the following steps before TALO can successfully perform update operations:
+If you deployed ZTP in a disconnected environment as instructed in the "Preparing the disconnected environment" section in "Deploying distributed units at scale in a disconnected environment", you need to do the following steps before TALO can successfully perform update operations:
 
 TALO can perform both platform and Operator updates:
 
 * For platform updates, you have to do the following:
 +
-. Mirror the desired {product-title} image repository.
-+ 
-For more information, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#installation-mirror-repository_ztp-deploying-disconnected[Mirroring the OpenShift Container Platform image repository]
+. Mirror the desired {product-title} image repository. For more information about how to mirror an image repository, see the "Mirroring the OpenShift Container Platform image repository" section.
 +
-. Prepare the update graph. You can do this the following ways:
-.. Use the OpenShift Update Service.
+[IMPORTANT]
+====
+The `imageContentSources` section, which can be retrieved with the `oc adm release mirror` command, needs to be saved for platform updates.
+For example, this is the output of the `imageContentSources` which contains the information of your mirrors:
 +
-For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#upgrading-disconnected-clusters[Upgrading disconnected clusters].
+[source,terminal]
+----
+imageContentSources:
```

Comment:
The out put must include two parts. Usually something like below (change mirror names please). Also please fix the indentation in the example.

```
imageContentSources:
 - mirrors:
   - mirror-ocp-registry.ibmcloud.io.cpak:5000/openshift-release-dev/openshift4
   source: quay.io/openshift-release-dev/ocp-release
 - mirrors:
   - mirror-ocp-registry.ibmcloud.io.cpak:5000/openshift-release-dev/openshift4
   source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
```